### PR TITLE
Fix for #1 and couple more minor improvements and bug fixes

### PR DESCRIPTION
**Pull request by markelog**

### CODE REVIEW

File: src/elements/Element.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ export default class Element {
      */
     get previousToken() {
         if (this._previousSibling) {
-            return this._previousSibling.firstToken;
+            return this._previousSibling.lastToken;
```
Comment (markelog) [inline src/elements/Element.js]: For second there, i thought i was loosing my mind, pretty deep error :-)

File: src/elements/Element.js [hunk 2]
```diff
@@ -298,6 +298,23 @@ export default class Element {
         return this._childElements.length;
     }
 
+    /**
+     * True if Element is a whitespace Token.
```
Comment (mdevils) [inline src/elements/Element.js]: Fix description, please :)

File: src/elements/Element.js (hunk 1 above)
Comment (mdevils) [inline src/elements/Element.js]: I can imagine :))))

**Comment by mdevils** [comment]
Amazing job! Thank you!

File: src/elements/Element.js (hunk 2 above)
Comment (markelog) [inline src/elements/Element.js]: Done

**Comment by mdevils** [comment]
Merged.

**Pull request merged**